### ISSUE TITLE
_ referenced before assignment

### ISSUE THREAD

**Issue by palewire** [jupyter-server/jupyter_server#431]
```python
Traceback (most recent call last):
  File "/home/palewire/.local/share/virtualenvs/california-coronavirus-scrapers-s_ZEjlcw/bin/jupyter-lab", line 8, in <module>
    sys.exit(main())
  File "/home/palewire/.local/share/virtualenvs/california-coronavirus-scrapers-s_ZEjlcw/lib/python3.8/site-packages/jupyter_server/extension/application.py", line 520, in launch_instance
    serverapp.start()
  File "/home/palewire/.local/share/virtualenvs/california-coronavirus-scrapers-s_ZEjlcw/lib/python3.8/site-packages/jupyter_server/serverapp.py", line 2100, in start
    self.start_app()
  File "/home/palewire/.local/share/virtualenvs/california-coronavirus-scrapers-s_ZEjlcw/lib/python3.8/site-packages/jupyter_server/serverapp.py", line 2059, in start_app
    self.launch_browser()
  File "/home/palewire/.local/share/virtualenvs/california-coronavirus-scrapers-s_ZEjlcw/lib/python3.8/site-packages/jupyter_server/serverapp.py", line 2020, in launch_browser
    self.log.warning(_('No web browser found: %s.') % e)
UnboundLocalError: local variable '_' referenced before assignment
```

**Comment by kevin-bates** [comment]
Thanks for the issue @palewire.  Please update `jupyter_server` to `1.4.1` as this issue was fixed via #421.

**Comment by palewire** [comment]
Thank you.
